model/bin/switch_meshcap

```diff
@@ -1 +1 @@
-UWM NCO NOGRB DIST MPI OMPG OMPH PR3 UQ FLX0 SEED ST4 STAB0 NL1 BT1 DB1 MLIM FLD2 TR0 BS0 RWND WNX1 WNT1 CRX1 CRT1 O0 O1 O2 O3 O4 O5 O6 O7 O14 O15 IC0 IS0 REF0
```

Comment:
Thanks for the explanation.  We do need that switch for the pre and post processing jobs in the workflow and typically expect it to be in this switch file.  I will test to see if I can add it in the workflow without causing issues.

Reply:
I'm not sure how the ww3 codes would be compiled in UFS since they're not in the ``cmake/src_list.cmake``

Reply:
This is ready to start on the UFS side, but it needs pre-approval first for the WW3 PR as a subcomponent. I can revert the removal of the NCO in the switch file if you require it.

Reply:
@DeniseWorthen the testing on my end has gone a ltitle slow -- apologies.  My run from today ran out of memory.  If you do not want to wait for my testing, please revert to add back NCO. My preference is to revert even if this will work to add this in the global-workflow as it's just nice to have all the switches that we will need there and not have to add/subtract in other bulid scripts.  That being said, I'm continuing my tests and will keep you posted.